spdx-utils/src/test/kotlin/SpdxDocumentModelTest.kt

```diff
@@ -53,15 +53,15 @@ private fun readResourceAsText(resourceFile: String): String =
 class SpdxDocumentModelTest : WordSpec({
     "The official YAML example from the SPDX specification version 2.2" should {
         "be deserializable" {
-            val yaml = readResourceAsText("/spdx-spec-examples/SPDXYAMLExample-2.2.spdx.yaml")
+            val yaml = readResourceAsText("/spdx-spec-examples/SPDXYAMLExample-2.2.spdx.yml")
```

Comment:
In contrast to what @tsteenbe said I'd argue that you should drop this change. If the original file was called `SPDXYAMLExample-2.2.spdx.yaml` as this is an exact copy, we should also keep the exact name. And it's a nice show-case that we can also handle the `.yaml` extension correctly.

Reply:
The files renamed here were anyway added by @fviernau and not by @neubs-bsi AFAICS.

Reply:
And if we agree to keep the `.yaml` extensino for YAML files in the `spdx-spec-examples` directory, then also the `SPDXYAMLExample-2.2-no-relationship-describes-or-document-describes.spdx.yml` file added in the frist commit should be renamed.